processing/src/test/java/org/apache/druid/query/DruidProcessingConfigTest.java

```diff
@@ -133,14 +134,11 @@ public void testInvalidSizeBytes()
         HEAP_SIZE,
         props
     );
-    Throwable t = Assert.assertThrows(
-        ProvisionException.class,
-        () -> injector.getInstance(DruidProcessingConfig.class)
-    );
-    Assert.assertTrue(
-        t.getMessage()
-         .contains("Cannot construct instance of `org.apache.druid.java.util.common.HumanReadableBytes`, problem: Invalid format of number: -1. Negative value is not allowed.")
-    );
+
+    ExceptionMatcher.of(ProvisionException.class)
+        .expectMessageContains(
+            "Cannot construct instance of `HumanReadableBytes`, problem: Invalid format of number: -1. Negative value is not allowed."
+        ).assertThrowsAndMatches(() -> injector.getInstance(DruidProcessingConfig.class));
```

Comment:
For a consistent formatting style:

```suggestion
    ExceptionMatcher
        .of(ProvisionException.class)
        .expectMessageContains(
            "Cannot construct instance of `HumanReadableBytes`, problem: Invalid format of number: -1. Negative value is not allowed."
        )
        .assertThrowsAndMatches(() -> injector.getInstance(DruidProcessingConfig.class));
```